docs/source/setup/sample.rst

```diff
@@ -228,3 +230,16 @@ To view the logs, run:
    ./orbit.sh -p -m tensorboard.main --logdir=logs
 
 .. _Tensorboard: https://www.tensorflow.org/tensorboard
+
+
+.. note::
+
+   In some examples above the ``--checkpoint`` argument is used.
+   It is not required: if not specified, the latest ``model.pt`` is taken automatically.
+
+   Example usage: ``--checkpoint model_999.pt``
+
+   You can also specify the run where to take the checkpoint from, for example: 
+   ``--load_run 2024-03-11_16-11-38``
+
+   The model checkpoints are saved and loaded from the ``logs`` directory (see above).
```

Comment:
This is pretty vague IMO. We only have load_run for RSL-RL case. For others this is misleading/unclear